op-dispute-mon/README.md

```diff
@@ -16,5 +16,14 @@ This will build the `op-dispute-mon` binary which can be run with
 ## Usage
 
 `op-dispute-mon` is configurable via command line flags and environment variables. The help menu
-shows the available config options and can be accessed by running `./op-dispute-mon --help`.
+shows the available config options and can be accessed by running `./bin/op-dispute-mon --help`.
 
+```shell
+
+# Start the op-dispute-mon with specified game factory address and RPC endpoints
+./bin/op-dispute-mon \
+  --game-factory-address <Game-Factory-Address> \
```

Comment:
Probably better to make the example use the `--network` flag which automatically provides the correct factory address for that network.